Add 10.13.1 release notes (#4882)
owncloud/docs: modules/ROOT/pages/server_release_notes.adoc

:page-aliases: {latest-server-version}@server:admin_manual:whats_new_admin.adoc, \
{latest-server-version}@server:ROOT:server_release_notes.adoc

> * xref:changes-in-10-13-0[Changes in 10.13.1]

Reviewer: ```suggestion
* xref:changes-in-10-13-1[Changes in 10.13.1]
```

* xref:changes-in-8-0[Changes in 8.0]
* xref:changes-in-7-0[Changes in 7.0]

== Changes in 10.13.1

> Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.0 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.

Reviewer: ```suggestion
Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
```

=== Open in Web Improvements ===

The "Open in Web" feature for direct editing of office documents now properly works on iOS and Desktop clients with Collabora. 

> === Avoid Loading 3d-party Resources

Reviewer: ```suggestion
=== Avoid Loading 3rd-party Resources
```

The "Open in Web" configuration offers icons to be shown in the clients. This is e.g. used by the iOS 12.0.3 client. We now ship the icons embedded in core.

=== Notable Changes

* Fix: disallow pre-signed url access if the signing key is not initialized.
* New code was added to dismiss invalid settings of the redirection endpoint URI as seen in the OAuth2 protocol, according to RFC#7636.

=== Updated App Versions

> Find below a list of updated apps in comparison with the 10.12.2 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.

Reviewer: ```suggestion
Find below a list of updated apps in comparison with the 10.13.0 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
```

* https://marketplace.owncloud.com/apps/graphapi [Graph API]: v0.3.1

> * https://marketplace.owncloud.com/apps/guests [Guests] 0.12.3

Reviewer: Is the change for whitelisted apps going to be in guests 0.12.3 and bundled with 10.13.1 ?

Author: yes.

* https://marketplace.owncloud.com/apps/oauth2 [OAuth2]: v0.6.1

=== Known Issues

> * Two cron jobs related to cleaning up left over thumbnails were not properly enabled in the past (`PreviewCleanupJob`). In 10.13.0 the code was fixed, and the jobs startd running. However, in certain cases with an exceptional high number of potential cleanups, these jobs can now take up a substantial amount of database resources. We propose to temporarily disable the offending job until we have a fix.

Reviewer: ```suggestion
* Two cron jobs related to cleaning up left over thumbnails were not properly enabled in the past (`PreviewCleanupJob`). In 10.13.0 the code was fixed, and the jobs started running. However, in certain cases with an exceptionally high number of potential cleanups, these jobs can now take up a substantial amount of database resources. We propose to temporarily disable the offending job until we have a fix.
```
